src/main/java/Exercise20.java

```diff
@@ -0,0 +1,17 @@
+public class Exercise20 {
+    public int happyNumber(int n){
+        int loop = 100;
+        while (loop>0){
```

Comment:
Cái này tương đương với

```suggestion
        for(int loop = 100;i>0; i--)
```

Reply:
Em cũng không biết khi nào nó sẽ dừng lại
Nên em nghĩ số 100 đủ lớn